Text/Parsec/Combinator.hs

```diff
@@ -74,6 +75,11 @@ between :: (Stream s m t) => ParsecT s u m open -> ParsecT s u m close
 between open close p
                     = do{ _ <- open; x <- p; _ <- close; return x }
 
+-- | @one p@ applies the parser @p@ once and returns it's result as
```

Comment:
Note, this is to make things like this easy:

`(one $ char 'a') <> (many $ char 'b')`

meaning, "one 'a', followed by many 'b', all in the result String"